compiler/src/iree/compiler/Codegen/SPIRV/test/config_broadcom_matmul.mlir

```diff
@@ -0,0 +1,359 @@
+// RUN: iree-opt --split-input-file --iree-gpu-test-target=videocore_vii --pass-pipeline='builtin.module(iree-spirv-select-lowering-strategy-pass)' %s | FileCheck %s
+
```

Comment:
Could you create a ticket to benchmark the different cases mentioned in this file, please?